docs/book/src/clusterctl/commands/upgrade.md

```diff
@@ -75,6 +75,16 @@ clusterctl upgrade apply \
 
 <aside class="note warning">
 
+<h1>Skip-level upgrades</h1>
+
+Cluster API only tests clusterctl upgrades from the latest minor version of each contract for a specific version 
+(e.g. v0.3 (v1alpha3) => v1.3, v0.4 (v1alpha4) => v1.3 and v1.2 (v1beta1) => v1.3). While we are not testing skip-level 
+upgrades from earlier versions (e.g. v1.1 => v1.3), it should work in general.
```

Comment:
> it should work in general

I wonder if this sounds a bit too vague, but I don't have better suggestions. Maybe others can weigh in?

Reply:
@CecileRobertMichon Kilian suggested an improvement. Does that sound better to you?

Reply:
+1 to the second version
Wondering if we can simplify the first sentence by just listing what upgrade path we are currently testing; also, if I'm not wrong we are missing 1.2 --> 1.3

Reply:
We can add an explicit list, but that also means we have to keep the list up-to-date.

I would prefer writing it more generic than that to avoid this